### DISCUSSION TITLE
Welcome to openship discussions!

### DISCUSSION THREAD

**Discussion by junaid33** [openshiporg/openship#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tosha31** [comment]
Are you still active/maintaining this repo? I would like to get involved if possible.

**Reply by junaid33** [reply]
Yes, I added the Admin UI at /dashboard a few months back. Overall, It’s stable and being used in production. In what capacity would you like to help? PRs are being accepted.

**Comment by dwdonline** [comment]
Hello @junaid33 
We are going to be using openship, and I'm hoping to get involved and contribute back. We we will be adding our (Acumatica), WMS (Savant and Logiwa), shopping carts (WooCommerce, MedusaJS, Magento), some drop-ship, notifications (slack, email, etc.), etc. Would love to have a conversation how my team can contribute back not only our code and get involved in the main project, and align our development and support with yours.
Philip

**Reply by junaid33** [reply]
Hey Philip, 
That sounds like a great idea and contributions to Openship are always welcome! Can you book a slot here and we can discuss: https://book.openship.org/junaid/30min?
